Containers/borgbackup/backupscript.sh

```diff
@@ -34,19 +34,23 @@ for volume in "${DEFAULT_VOLUMES[@]}"; do
 done
 
 # Check if target is mountpoint
-if ! mountpoint -q /mnt/borgbackup; then
-    echo "/mnt/borgbackup is not a mountpoint which is not allowed."
+if [ -z "$BORG_REMOTE_REPO" ] && ! mountpoint -q "$MOUNT_DIR"; then
+    echo "$MOUNT_DIR is not a mountpoint which is not allowed."
     exit 1
 fi
 
-# Check if target is empty
-if [ "$BORG_MODE" != backup ] && [ "$BORG_MODE" != test ] && ! [ -f "$BORG_BACKUP_DIRECTORY/config" ]; then
-    echo "The repository is empty. Cannot perform check or restore."
+# Check if repo is uninitialized
+if [ "$BORG_MODE" != backup ] && [ "$BORG_MODE" != test ] && ! borg info; then
```

Comment:
Honestly, I would like to keep these config checks because what if the locla target drive is not mounted anymore? Then it will automatically create a new archive...

Same in all other places

```suggestion
if [ "$BORG_MODE" != backup ] && [ "$BORG_MODE" != test ] && ! [ -f "$BORG_BACKUP_DIRECTORY/config" ]; then
```

So you will probably need to do this check differently for local and remote...

Reply:
If it's not mounted, `borg info` would fail unless there was already a valid borg repo in the unmounted directory. So I believe this works the same way as checking for the config file except that the former also works for remote.

```
❯ borg info .  # exits nonzero
/home/user/fjj is not a valid repository. Check repo config.

❯ borg info doesnotexist  # exits nonzero
Repository /home/user/fjj/doesnotexist does not exist.
```

> Then it will automatically create a new archive

I assume you mean new repo? The presence of `/nextcloud_aio_volumes/nextcloud_aio_mastercontainer/data/borg.config` will prevent initializing a new repo https://github.com/nextcloud/all-in-one/pull/4804/files/a25ebd7a60370c3f0999034fbe0d284982864535#diff-95fc189cd306db6f940ec73d973cf7b36276bff075d61dff20e17c3dcf295dabR122
which in the remote case is an empty file I create https://github.com/nextcloud/all-in-one/pull/4804/files/a25ebd7a60370c3f0999034fbe0d284982864535#diff-95fc189cd306db6f940ec73d973cf7b36276bff075d61dff20e17c3dcf295dabR168